Week2/homework/maartjes_work.js

```diff
@@ -45,3 +45,30 @@ const tuesday = [
 const tasks = monday.concat(tuesday);
 
 // Add your code here
+ let durationInHours = tasks.map(hours => hours.duration / 60 ); 
+ console.log(durationInHours);
+
+ // (9) [3, 2, 0.3333333333333333, 3.3333333333333335, 4, 3,
+ // 0.16666666666666666, 3.3333333333333335, 0.6666666666666666]
+
+ let moreThanTwo = durationInHours.filter(hours => hours > 2 );
+ console.log(moreThanTwo);
+
+//(5) [3, 3.3333333333333335, 4, 3, 3.3333333333333335] 
+
+let salary = moreThanTwo.map(hour => hour * 10 );
+
+let amount = salary.reduce((getsum , total) => getsum + total );
+console.log(amount);
+
+//166.66666666666669
+
+let theSalary = '€ ' +  amount.toFixed(2);
```

Comment:
I am not sure if this has been brought up in a class by a teacher before but if you wish to do a concatenation like this, you can explore template literals - https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Template_literals

Could you try to do the same please? 🙂